.tools/test/sqs_lambda_to_batch_fargate/lambda/export_logs.py

```diff
@@ -19,17 +19,15 @@ def handler(event, context):
     logger.debug(f"BUCKET_NAME: {os.environ['BUCKET_NAME']}")
     logger.debug(f"INCOMING EVENT: {event}")
 
-    status = event["detail"]["status"]
-
     if "Batch Job State Change" not in event["detail-type"]:
         logger.info(f"Non-triggering Batch event: {event['detail-type']}")
         return
-    if "TIMED_OUT" in status:
+    if "TIMED_OUT" in event["detail"]["status"]:
```

Comment:
Why did you "undo" the extract status as constant?

Reply:
It causes errors because there are events without a status that hit this line of code. I could have moved this down 2 lines (after the detail-type: Batch Job State Change) but opted to revert the setting of the variable, as it only appears a few times.

Happy to add it back if you feel strongly stylistically, but this is low impact and does not improve the performance of the code in any meaningful way.

Reply:
Honestly I'm surprised not every event has a status, that's the real news to me